### ISSUE TITLE
[Platform] GCP SSH key for yugabyte is not attached to isntances

### ISSUE THREAD

**Issue by skorobogatydmitry** [yugabyte/yugabyte-db#7454]
On reacting universe Platform provision instances and "manually" adds SSH keys to authorized for yugabyte user.
After some time GCP realize this and remove these keys so all next tasks fails.

**Comment by streddy-yb** [comment]
hi @skorobogatydmitry - This is not in Platform's control. You have to disable Google Accounts Daemon to workaround this issue and this needs to be documented. See https://stackoverflow.com/questions/53543631/ssh-keys-keep-getting-deleted-from-google-compute-engine-vm for more details. Can you try disabling the daemon and see if it addresses the problem? 
cc @jharveysmith

**Comment by skorobogatydmitry** [comment]
@streddy-yb , yes, I know that GCP overrides `authorized_keys`. But we can add key to instance through Google Cloud SDK without need to have custom AMI or any manual steps to fix that.
